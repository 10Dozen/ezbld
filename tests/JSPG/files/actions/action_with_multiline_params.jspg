@ Action with multiline params || MyTag

*scene: CurrentScene
*icon: {text: 'A', class: 'icon-button-red'}
**goto:
    Foo.Bar()
    return Foo.GetNextScene()
**condition:
    if (Foo.Bar()) return true
    return Foo.CheckOtherCondition()
**exec:
    Player.AddXP(1000)
    Player.AddMoney(500)
    GUI.UpdatePlayerInfo()
