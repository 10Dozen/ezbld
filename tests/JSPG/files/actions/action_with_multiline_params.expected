[0]
cls = JSPGAction
name = Action with multiline params
tag = MyTag
scene = CurrentScene
icon = {text: 'A', class: 'icon-button-red'}
goto =
    \tFoo.Bar()
    \treturn Foo.GetNextScene()
condition =
    \tif (Foo.Bar()) return true
    \treturn Foo.CheckOtherCondition()
exec =
    \tPlayer.AddXP(1000)
    \tPlayer.AddMoney(500)
    \tGUI.UpdatePlayerInfo()
